Trading/Exchange/bybit/bybit_exchange.py

```diff
@@ -0,0 +1,36 @@
+#  Drakkar-Software OctoBot-Tentacles
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_trading.exchanges as exchanges
+import octobot_trading.errors
+
+
+class Bybit(exchanges.SpotCCXTExchange):
+    DESCRIPTION = ""
+
+    @classmethod
+    def get_name(cls) -> str:
+        return 'bybit'
+
+    @classmethod
+    def is_supporting_exchange(cls, exchange_candidate_name) -> bool:
+        return cls.get_name() == exchange_candidate_name
+
+    async def get_symbol_prices(self, symbol, time_frame, limit: int = 200, **kwargs: dict):
+        try:
+            candles = await self.connector.client.fetch_ohlcv(symbol, time_frame.value, limit=limit, params=kwargs)
+            return candles
```

Comment:
If we do nothing with the candles variable, I think we should return the awaited result directly

Reply:
exact !